client/lib/domains/constants.js

```diff
@@ -17,7 +17,40 @@ const registrar = {
 	MAINTENANCE: 'Registrar TLD Maintenance'
 };
 
+const tlds = {
```

Comment:
I'm not really sure what you meant @Tug ,
if we need to add a new gTld, lets say `.yury`, I'll need to figure out after adding the product, that I need to come here ( how would I know that ? ) and update that list.

IMO, creating a convention or an additional field is the approach we should pursue .